includes/pages/user_settings.php

```diff
@@ -19,7 +19,7 @@ function user_settings() {
   $mail = $user['email'];
   $email_shiftinfo = $user['email_shiftinfo'];
   $jabber = $user['jabber'];
-  $hometown = $user['Hometown'];
+  $kommentar = $user['kommentar'];
```

Comment:
I don't think we should change it there then but nonetheless keep it to English as far as possible. The variable doesn't need to be called `kommentar`, also the other occurrences not directly referencing the database.